lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2612,6 +2628,31 @@ def _add_to_dict(metadata):
         return d
 
 
+def loggly_time_string(dt):
+    r = dt.strftime("%Y-%m-%d") + "T" + dt.strftime("%H") + "%3A" + dt.strftime("%M") + "%3A"
+    r += str(dt.second) + str(round(dt.microsecond * (10.0**-5), 3)) + "Z"
+    return r
+
+
+def get_loggly_query_string(lbry_id):
+    decoded_id = base58.b58encode(lbry_id)
+    now = datetime.now()
+    yesterday = now.replace(day=now.day - 1)
+    loggly_params = (decoded_id[:SHORT_ID_LEN], loggly_time_string(yesterday), loggly_time_string(now))
+    loggly_search_url = "https://lbry.loggly.com/search#terms=json.lbry_id%3A"
+    query_string = "%s*&from=%s&until=%s&source_group=" % loggly_params
+    return loggly_search_url + query_string
+
+
+def report_bug_to_slack(message, lbry_id, platform_name, app_version):
+    webhook = "mXkfdToSwHy2QseofC1GPdMxRZfDT9HDx33MkASV7FfQvcqvjEbd14M3tyhoqtepQRgMnmg3Pr59BEFvpx4vYHsKhbEXTC6m2qjX7ow8v"
```

Comment:
add this to conf.settings (similar to the analytics token) and use utils.obfuscate / deobfuscate for consistency.